### PR TITLE
Working on modal window

### PR DESCRIPTION
**Pull request by doubt-ur-vibe**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -43,8 +43,8 @@
             <label>Brand
```
[the remark this answers is not in the document]
Comment (doubt-ur-vibe) [reply index.php:44]: Oh yeah, good shout. Thanks!

File: index.php [hunk 2]
```diff
@@ -43,8 +43,8 @@
...
             <input type="text"/>
             </label>
-            <label>Limited Edition
-            <input type="radio"/>
+            <label class="checkbox">Limited Edition
+            <input type="checkbox"/>
             </label>
             <label>Price
             <input type="number"/>
```
[the remark this answers is not in the document]
Comment (doubt-ur-vibe) [reply index.php]: Cool, I changed it to a range. Thanks a lot! :)

File: data.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+require_once 'function.php';
+function addFormToDataBase() {
```
Comment (ashleycoles) [inline data.php:3]: Missing doc block!

File: edit.php [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?php
+    require_once 'data.php';
+    addFormToDataBase();
+    header("Location: index.php");
```
Comment (ashleycoles) [inline edit.php:4]: I'd actually suggest moving this header into the addFormToDataBase function

File: data.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    if(isset($_POST['name']) && isset($_POST['img']) && isset($_POST['brand']) && isset($_POST['price'])) {
+        $nameInput = $_POST['name'];
+        $imgInput = $_POST['img'];
+        $brandInput = $_POST['brand'];
+        $limitedInput = isset($_POST['limited']) ? 1 : 0;
+        $priceInput = $_POST['price'];
```
Comment (ashleycoles) [inline data.php:9]: Definitely want to add some basic validation here, for example making sure that the name input isn't empty (I managed to add a nameless deck of cards by just typing spaces in.

I'd run all of your string based inputs through `trim()` and `htmlspecialchars()` before making sure they actually contain characters.

One thing you could do is add error messages into the header redirect which you could then display on the index.php file.

Something like `header('Location: index.php?error=Name must contain more than 3 characters');`

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Awesome work Ali, works a treat until I start being mean and abusing the form! The comments I've added are all around making the form more robust

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**